Formula/frege-repl.rb

```diff
@@ -3,7 +3,7 @@ class FregeRepl < Formula
   homepage "https://github.com/Frege/frege-repl"
   url "https://github.com/Frege/frege-repl/releases/download/1.4-SNAPSHOT/frege-repl-1.4-SNAPSHOT.zip"
   version "1.4-SNAPSHOT"
-  sha256 "2cf1c2a8f7b64c9d70b21fbfd25b2af3f5e3bebe3662f724afd435d01bddafec"
+  sha256 "2ca5f13bc5efaf8515381e8cdf99b4d4017264a462a30366a873cb54cc4f4640"
```

Comment:
Please add `revision 1` on a new line right underneath `sha256` and then this is good to go.